conbench/tests/api/_expected_docs.py

```diff
@@ -1027,7 +1027,7 @@
                     },
                     "stats": {"$ref": "#/components/schemas/BenchmarkResultCreate"},
                     "tags": {
-                        "description": 'The set of key/value pairs that defines a benchmark case permutation. Top-level keys must be strings. Allowed value types are ... (to be specified).  The special key "name" must be provided with a string value: it indicates the name of the conceptual benchmark that was performed for obtaining the result at hand. All case permutations of a conceptual benchmark by definition have this name in common.  We advise that each unique case (as defined by the complete set of key/value pairs) indeed corresponds to unique benchmarking behavior. That is, typically, these key/value pairs directly correspond to input parameters to your benchmarking method.  Example: a conceptual benchmark with name "foo-write-file" might have meaningful case permutations involving tag names such as `compression_method` (values: `gzip`, `lzma`, ...), `file_format` (values: `csv`, `hdf5`, ...), `dataset_name` (values: `foo`, `bar`, ...). For each conceptual benchmark, it is OK to have no or many case permutations.',
+                        "description": 'The set of key/value pairs that defines a benchmark case permutation. Top-level keys must be strings. Values must be strings, too. Keys with empty string values are ignored.  The special key "name" must be provided with a string value: it indicates the name of the conceptual benchmark that was performed for obtaining the result at hand. All case permutations of a conceptual benchmark by definition have this name in common.  We advise that each unique case (as defined by the complete set of key/value pairs) indeed corresponds to unique benchmarking behavior. That is, typically, these key/value pairs directly correspond to input parameters to your benchmarking method.  Example: a conceptual benchmark with name "foo-write-file" might have meaningful case permutations involving tag names such as `compression_method` (values: `gzip`, `lzma`, ...), `file_format` (values: `csv`, `hdf5`, ...), `dataset_name` (values: `foo`, `bar`, ...). For each conceptual benchmark, it is OK to have no or many case permutations.',
```

Comment:
```suggestion
                        "description": 'The set of key/value pairs that defines a benchmark case permutation. Top-level keys must be strings. Values must be strings, too. Keys with empty string values are ignored.  The special key "name" must be provided with a string value: it indicates the name of the conceptual benchmark that was performed for obtaining the result at hand. All case permutations of a conceptual benchmark by definition have this name in common.  We advise that each unique case (as defined by the complete set of key/value pairs) indeed corresponds to unique benchmarking behavior. That is, typically, these key/value pairs directly correspond to input parameters to your benchmarking method.  Example: a conceptual benchmark with name "foo-write-file" might have meaningful case permutations involving tag names such as `compression_method` (values: `gzip`, `lzma`, ...), `file_format` (values: `csv`, `hdf5`, ...), `dataset_name` (values: `foo`, `bar`, ...). For each conceptual benchmark, it is OK to have no or many tags which would result in one or many case permutations (if you supply no tags, there is necessarily a single mutation with a special property that it has no tags).',
```

Ok to clean this up here? I'm open to other wording suggestions of course, but do think it would be good to be explicit here.

Reply:
Ah, I now saw what you edited. This is about https://github.com/conbench/conbench/pull/936#discussion_r1145988615.

Reply:
Cool. So, I added the change to #954 where we also needed to re-generate the `_expected_docs.py`.